### PR TITLE
feat: assignment of a prescription to a pharmacy

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/dispensary/views.py [hunk 1]
```diff
@@ -30,9 +30,11 @@ def update(self, request, pk=None):
         prescription = get_object_or_404(Prescription, pk=pk)
         serializer = PrescriptionSerializer(prescription, data=request.data, partial=True)
         serializer.is_valid(raise_exception=True)
+        if 'pharmacy' in request.data:
```
[the remark this answers is not in the document]
Comment (Kavindu-248) [reply apps/dispensary/views.py:33]: @Raam124  over there  that feature is associating a prescription to a pharmacy is the feature , or have I misinterpreted.

[the remark this answers is not in the document]
Comment (Raam124) [reply apps/dispensary/views.py:33]: Recheck with the scenario

File: apps/assesment/serializers.py
[the remark this answers is not in the document]
Comment (Kavindu-248) [reply apps/assesment/serializers.py:1]: @Raam124  I thought the missing link was when creating an appointment or assesment the user can select the pharmacy , _" These prescriptions are sent to a pharmacy preselected by the patient at the creation of the video or form assessment order."_

[the remark this answers is not in the document]
Comment (Raam124) [reply apps/assesment/serializers.py:1]: understood

**Review by Raam124 — APPROVED** [review]

**Pull request merged**